src/js/components/deployments/deploymentitem.js

```diff
@@ -49,7 +49,11 @@ export const DeploymentRelease = props => {
     deployment: { artifact_name, type = DEPLOYMENT_TYPES.software }
   } = props;
   const deploymentRelease = type === DEPLOYMENT_TYPES.configuration ? type : artifact_name;
-  return <div key="DeploymentRelease">{deploymentRelease}</div>;
+  return (
+    <div className="text-overflow" key="DeploymentRelease" style={{ whiteSpace: 'initial' }}>
```

Comment:
Q: Why initial and not `normal`?

Reply:
to stick with the default, which might get modified by an outer component - setting it to normal would interfere with any inheritance, which in this case I don't intend to do, but rather just restore what the `text-overflow` class defines